lib/streamlit/error_util.py

```diff
@@ -14,14 +14,21 @@
 
 from __future__ import annotations
 
+import os
 from typing import Final
 
-import streamlit as st
+import streamlit
 from streamlit import config
+from streamlit.delta_generator_singletons import get_dg_singleton_instance
 from streamlit.errors import UncaughtAppException
 from streamlit.logger import get_logger
 
 _LOGGER: Final = get_logger(__name__)
+# The path to the streamlit package directory used to
+# suppress internal tracebacks in error messages.
```

Comment:
nit: I would add a sentence here that we use a string and not the module itself to avoid circular imports

Reply:
I changed it again since we can also just use the `streamlit` module there since this is imported anyway.

Reply:
@LukasMasuch previously only the scriptrunner module was excluded. Is it correct to exclude the whole streamlit module?

Reply:
Yep, it even improves thi since we don't want to show any Streamlit internal code in the rich traceback logs. It will still show the line numbers of internal files but not the surrounding code.

Reply:
Error looks like this: 

<img width="627" alt="image" src="https://github.com/user-attachments/assets/4821409b-3ab5-4386-b380-3eaea43e1e0b">